common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1059,7 +1059,7 @@ def get_item(self, usage_key, depth=0, **kwargs):
                 log.debug("Found more than one item for '{}'".format(usage_key))
             return items[0]
 
-    def get_items(self, course_locator, settings=None, content=None, qualifiers=None, **kwargs):
+    def get_items(self, course_locator, settings=None, content=None, qualifiers=None, in_tree_only=False, **kwargs):
```

Comment:
After some thought, I'd prefer that this flag were called `include_orphans` and defaulted to `True`. The definition of an orphan is a block that exists in the course block list but not in the tree. Mentioning orphans in the flag is more direct and speaks directly to why the flag exists. And multiple other places in the code refer to orphans already. `in_tree_only` obscures the fact that the flag exists only because orphans exist.